x/auth/client/rest/query.go

```diff
@@ -49,7 +49,7 @@ func QueryAccountRequestHandlerFn(
 
 		// the query will return empty if there is no data for this account
 		if len(res) == 0 {
-			w.WriteHeader(http.StatusNoContent)
+			w.WriteHeader(http.StatusOK)
```

Comment:
ditto, we need to test that this returns an empty `Account`

Reply:
Added test which covers it `TestAccountBalanceQuery `